docker/run-uc/Dockerfile

```diff
@@ -0,0 +1,108 @@
+FROM eclipse-temurin:17-jre-alpine AS run_uc
```

Comment:
It looks like this one isn't available for the ARM64 architecture: https://hub.docker.com/layers/library/eclipse-temurin/17-jre-alpine/images/sha256-5f6c6ca67001cf84c9b5a1521d17fea819f703d62fffa4de919993b81293f812?context=explore 😭

Reply:
This is easy, should be solved now.